docs/dumping_transformation.rst

```diff
@@ -0,0 +1,41 @@
+Dumping a ``Transformation`` to JSON
+====================================
+
+If you're trying to run a full campaign of simulations representing an
+alchemical network, we generally recommend saving objects using our storage
+tools, when avoids saving duplicate information to disk.
+
+.. TODO: add links to storage tools once they're complete
+
+However, there are situations where it is reasonable to serialize a single
+:class:`.Transformation`. For example, this can be useful when trying to
+compare results run on different machines. This also provides a trivial way
+for a user to run edges in parallel, if they don't want to use the more
+sophisticated techniques we have developed.
+
+For these cases, we have made it very easy for a user to dump a
+transformation to JSON. Simply use the method
+:meth:`.Transformation.dump`. This can take a filename (pathlike) or an
+already-opened file object. For example:
+
+.. code::
+
+    transformation.dump("mytransformation.json")
+
+Be aware this this is not designed to be space-efficient. That is, if you
+have the same object in memory used in two location (for example, the same
+``ProteinComponent`` in both ``ChemicalSystems``, then 
```

Comment:
then.... ?

I think it was all these warnings about space efficiency that made me think of making the canonical dump format `.json.gz` as LZ compressions will eat the repeated data for breakfast

Reply:
Fixed the hanging then.... 

If I dump a network containing a single protein and 1000 edges as JSON representations of transformations, I get 2000 copies of the protein. If I dump it as gzipped transformations, I get 1000 copies minus reduction by gzip (still 1000 copies of what gzip produced, though). If I dump it using our better serialization, I get 1 copy (possibly gzipped). Now who is eating breakfast?